third_party/Makefile

```diff
@@ -4,15 +4,24 @@
 
 SB_MICRO_PATH ?= "/usr/local"
 
-.PHONY: all cutlass
+.PHONY: all cutlass bandwidthTest
 
 # Build all targets.
-all: cutlass
+all: cutlass bandwidthTest
 
+# Create $(SB_MICRO_PATH)/bin and  $(SB_MICRO_PATH)/lib if not exist.
+sb_micro_path:
+	if [ ! -d  $(SB_MICRO_PATH)/bin ]; then mkdir -p $(SB_MICRO_PATH)/bin; fi
+	if [ ! -d $(SB_MICRO_PATH)/lib ]; then mkdir -p $(SB_MICRO_PATH)/lib; fi
```

Comment:
Is it aligned with our design?

Reply:
Some thirdparty projects don't provide the 'make install' option, so we should create the target folder if it doesn't exist and move the binary to the target folder.
I think it's OK.